src/main/scala/io/iohk/ethereum/merklepatriciatrie/package.scala

```diff
@@ -0,0 +1,34 @@
+package io.iohk.ethereum
+
+package object merklepatriciatrie {
+  trait ByteArraySerializable[T] {
+    def toBytes(input: T): Array[Byte]
+
+    def fromBytes(bytes: Array[Byte]): T
+  }
+
+  trait DataSource {
```

Comment:
Yes, we decided to put them there as they should be commonly implemented in order to use the trie (if any our implementations of them are not used). Do you think it would be better to put them elsewhere?